activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+*   Active Support cache stores now support replacing the default compressor via
+    a `:compressor` option. The specified compressor must respond to `compress`
+    and `decompress`.  For example:
```

Comment:
I think `inflate` and `deflate` are much more common than `compress` and `decrompress` (even though I do prefer the later, or better `load/dump` as it's more composable).

`Snappy`, `Zlib`, and `Brotli` all expose `inflate/deflate`.

So it would make sense to standardize on that, we can make a PR to `zstd` to alias these methods.